### PR TITLE
Add gem badge and remove installation guide via Gemfile

### PR DESCRIPTION
**Pull request by splattael**
Preview https://github.com/ruby-formatter/rufo/blob/readme-badges/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +19,7 @@ Rufo supports all Ruby versions >= 2.2.4, but works most reliably with >= 2.3.**
 Add this line to your application's Gemfile:
 
 ```ruby
-gem 'rufo'
+gem 'rufo', '~> 0.1.0'
```
Comment (lucerion) [inline README.md]: Could you explain, please, why we force gem version?

Comment (splattael) [reply README.md]: Personally, I'd like to pin a gem version as much as possible.

I know that the default README template generated via `bundle gem NAME` does not specify a version. I do not like this. I rather prefer Elixir's approach and encourage users to pin versions.

Having said that, I do not think we need `gem 'rufo'` at all in `README` for `rufo`, do we?
It's just a tool which you normally install via `gem install rufo`, right?

Should we drop the following parts from the readme?

```
Add this line to your application's Gemfile:

gem 'rufo', '~> 0.1.0'
And then execute:

$ bundle
```

:question:

Comment (lucerion) [reply README.md]: I think we can remove Gemfile part because rufo is a standalone tool.

**Comment by splattael** [comment]
Installation guide via Gemfile removed from README as suggested by @lucerion.

Ready to merge :twisted_rightwards_arrows:

**Comment by lucerion** [comment]
@splattael thank you again!

**Comment by gingermusketeer** [comment]
I think we need to support including rufo in the gem file for two reasons: 
1. Pinning rufo to a specific version for a project. Not all projects will upgrade as quickly as others and depending on a global version will introduce unwanted formatting changes if different developers have different versions installed. With prettier I usually bump the version in a focused PR so formatting changes can be kept separate from logic changes. 
2. CI should check that the project has the expected formatting. The easiest way to do this is include it as a development dependency. Also for the reasons as above it should be the same version as is present on the developers machine. 

Thoughts?

**Comment by splattael** [comment]
@gingermusketeer You are absolutely right!

I did not think of using different `rufo` versions per project :disappointed: 

I will revert e3c5a5f? @lucerion, do you agree?

**Comment by lucerion** [comment]
@gingermusketeer good points! I thought about different cases of rufo usage and now I see it as external tool for personal usage, not formatting standard for project. About second point (just a question) - how I can integrate rufo with CI? I should wrote script that compare file with its formatted version? Maybe some CIs can do it, but it sounds like rufo feature :)

@splattael :+1:

**Comment by gingermusketeer** [comment]
@splattael that sounds great. The other component to this is that editor integrations need to prefer the gemfile version of rufo over any global version. I am not sure if this is the behaviour currently. 

Also should it be `gem 'rufo', require: false` as it is a executable rather than a library for use in an application?

@lucerion At the moment I am using rufo for personal projects as well, but I am very keen to start using it on the work related projects I have! I think I will start doing this once `0.2.0` is out.

RE CI I believe that you should be able to use the check option, `rufo --check .`. I do wonder if we can add a rake task so that you can do something like `gem 'rufo', require 'rufo/rake'` and have it add a hook to the test script so that it is easy to integrate into CI test runs.

**Comment by splattael** [comment]
:+1 about rake tasks in `lib`.

Note that we are already have some rake tasks in `rakelib/`. These should be extracted into `rufo/rake`. I agree!

**Pull request merged**